### ISSUE TITLE
Additional .factor-roots add new monitoring threads on every reload

### ISSUE THREAD

**Issue by AlexIljin** [factor/factor#1758]
**Steps to reproduce:**
```
{ "c:/windows" } home ".factor-roots" append-path utf8 set-file-lines
5 [ load-vocab-roots ] times
threads.
```
**Expected:**
Only one `Root monitor: "c:/windows" mailbox`, and only one `Windows monitor thread` waiting on it.
**Actual:**
Five of each.

**Comment by AlexIljin** [comment]
I made a fix to address this issue, see PR.

**Comment by mrjbq7** [comment]
Fixed in by #1763.
